server/controllers/documents.js

```diff
@@ -131,7 +129,7 @@ class DocumentController {
         document.content = req.body.content;
         document.save().then(() => {
           res.status(200).json({
-            msg: 'User updated'
+            msg: 'Document updated'
           });
         }).catch((err) => {
           res.status(500).json({
```

Comment:
internal server error?... you might want to look into the status code, I don't think its a 500 error